content/java/concepts/calendar/terms/isSet/isSet.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: '.isSet()'
+Description: 'Checks whether the given calendar field has a value set or not.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. This method returns `true` if the given calendar field has a value set, `false` otherwise. It returns true when the given parameter has been set by internal field interpretations. `isSet()` method is a non-static method, it is accessible with the class object.
+
+## Syntax
+
+```pseudo
+result = myCalendar.isSet(calendar_field)
+```
+
+Parameters: The method takes one parameter calndr_field that refers to the `calendar_field` which is to be operated upon.
+Return Value: The method either returns `true` if the value of the calendar field is set, else it returns `false`.
+
+> **Note:** Please note that the `Calendar` class has been replaced by the `LocalDate`, `LocalTime`, and `LocalDateTime` classes in the Java 8 and later Date and Time API. These classes provide a more modern and user-friendly API for working with dates and times in Java.
+
+## Example
+
+The following example shows the usage of `java.util.calendar.isSet()` method.
+
+```java
+import java.util.*;
```

Comment:
examples should used 2 spaces not tabs and comments should begin with a capital
// Create a calendar